source/recurring_payments/index.html.md.erb

```diff
@@ -182,7 +182,9 @@ Read more about creating payments in [our guidance for taking payments](/making_
 
 ### Update a user’s recurring payment card details
 
-You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card.
+You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card. 
+
+You could be charged failed fees if you try to take payments from an expired card.
```

Comment:
Is "failed fees" a common phrase? Is it "failed payment fees" or something else more descriptive? You'd know probably know way better than me, but just flagging I had to think twice.

Reply:
Agreed - will push a fix now.